doc/htmldoc/nest_sonata/nest_sonata_guide.rst

```diff
@@ -0,0 +1,259 @@
+.. _nest_sonata:
+
+NEST SONATA guide 
+=====================
+
+NEST supports building and simulating networks of point neurons described by the SONATA format [1]_. 
+This guide provides the details about how a SONATA network must be specified to be supported natively by NEST. 
+
+
+.. _sec:sonata_overview:
+
+Overview of the SONATA format 
+-----------------------------
+
+The SONATA (Scalable Open Network Architecture TemplAte) format provides a framework for storage and exchange of 
```

Comment:
```suggestion
The SONATA (Scalable Open Network Architecture Template) format provides a framework for storage and exchange of 
```

Reply:
Since SONATA is derived from **S**calable **O**pen **N**etwork **A**rchitecture **T**empl**A**te I think it makes sense to keep it as is. This is also in line with the spelling in the SONATA paper and documentation.

Reply:
Ah, sorry you're right :)